modules/nf-core/sentieon/readwriter/meta.yml

```
@@ -0,0 +1,71 @@
+name: sentieon_readwriter
+description: Merges BAM files, and/or convert them into cram files. Also, outputs the result of applying the Base Quality Score Recalibration to a file.
+keywords:
+  - merge
+  - convert
+  - readwriter
+  - sentieon
+tools:
+  - sentieon:
+      description: |
+        Sentieon® provides complete solutions for secondary DNA/RNA analysis for a variety of sequencing platforms, including short and long reads.
+        Our software improves upon BWA, STAR, Minimap2, GATK, HaplotypeCaller, Mutect, and Mutect2 based pipelines and is deployable on any generic-CPU-based computing system.
+      homepage: https://www.sentieon.com/
+      documentation: https://www.sentieon.com/
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
+        e.g. [ id:'test', single_end:false ]
+  - meta:
```

```suggestion
  - meta2:
```
Do you know the order in which these fields should be listed? I got the impression that it is standard to list them in the same order in the meta.yml input-section as the listen in the input-section in the main.nf.

---

I have seen the parameters listed in random order in some modules, so I don't think there is one. Or even if there is, I suppose its not strictly followed. 😄